webui/react/src/services/decoder.ts

```diff
@@ -464,25 +466,73 @@ export const mapSearchExperiment = (
   };
 };
 
-export const mapV1Experiment = (
+export function mapV1Experiment(
   data: Sdk.V1Experiment,
   jobSummary?: types.JobSummary,
-): types.ExperimentItem => {
+  config?: undefined,
+): types.BulkExperimentItem;
+export function mapV1Experiment(
+  data: Sdk.V1Experiment,
+  jobSummary?: types.JobSummary,
+  config?: Sdk.V1GetExperimentResponse['config'],
+): types.FullExperimentItem;
+export function mapV1Experiment(
+  data: Sdk.V1Experiment,
+  jobSummary?: types.JobSummary,
+  config?: Sdk.V1GetExperimentResponse['config'],
+): types.FullExperimentItem | types.BulkExperimentItem {
+  if (config === undefined) {
+    const bulkExpItem: types.BulkExperimentItem = {
+      archived: data.archived,
+      checkpoints: data.checkpointCount,
+      checkpointSize: parseInt(data?.checkpointSize || '0'),
+      description: data.description,
+      duration: data.duration,
+      endTime: data.endTime as unknown as string,
+      externalExperimentId: data.externalExperimentId,
+      externalTrialId: data.externalTrialId,
+      forkedFrom: data.forkedFrom,
+      hyperparameters: {},
```

Comment:
how do we get the hyperparameters from bulk if we return early here?

Reply:
this is why review is useful. completely missed this. fixed